js/bitmart.js

```diff
@@ -1816,9 +1818,17 @@ module.exports = class bitmart extends Exchange {
         if (ioc) {
             request['type'] = 'ioc';
         }
+        const marginMode = this.safeString (params, 'marginMode');
```

Comment:
@Dan-krm For the sake of consistency, can you please use the recently added `handleMarginModeAndParams` here ?